misc/images/materialized/entrypoint.sh

```diff
@@ -61,5 +61,6 @@ export MZ_ADAPTER_STASH_URL=${MZ_ADAPTER_STASH_URL:-postgresql://root@$(hostname
 export MZ_ORCHESTRATOR=${MZ_ORCHESTRATOR:-process}
 export MZ_ORCHESTRATOR_PROCESS_SECRETS_DIRECTORY=${MZ_ORCHESTRATOR_PROCESS_SECRETS_DIRECTORY:-/mzdata/secrets}
 export MZ_ORCHESTRATOR_PROCESS_SCRATCH_DIRECTORY=${MZ_ORCHESTRATOR_PROCESS_SCRATCH_DIRECTORY:-/scratch}
+export MZ_BOOTSTRAP_ROLE=${MZ_BOOTSTRAP_ROLE:-/materialize}
```

Comment:
Are you sure it's supposed to have the forward slash? I only see the forward slash used for arguments that are URLs, while this is supposed to be a role name.

Reply:
No, you are right. I will fix it.